spec/notifiers/publishing_api_notifier_spec.rb

```diff
@@ -9,7 +9,7 @@
       expect(Services.publishing_api).to receive(:put_content).with('69af22e0-da49-4810-9ee4-22b4666ac627', test: :payload)
       expect(Services.publishing_api).to receive(:publish).with('69af22e0-da49-4810-9ee4-22b4666ac627', 'minor')
```

Comment:
We should check here that `publishing_api` receives calls with more than just the original content id and that the payloads meet the requirements of a generic schema.